.github/workflows/publish.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish Package to npmjs
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
```

Comment:
How does this `matrix.node-version` thing work? From the GitHub Actions docs ([link](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs#specifying-the-nodejs-version)), it seem like we also need to specify:

```yaml
strategy:
  matrix:
    node-version: ['18.x', '20.x']
```

(Node 16.x is about to be EOL'd.)

Reply:
It looks like it will run the workflow on multiple versions of node.  I was just basing this on the previous workflow.  Will try using a matrix instead.